docs/api/source/index.rst

```diff
@@ -39,16 +39,16 @@ ship both `ESM <https://nodejs.org/docs/latest-v16.x/api/esm.html>`__ and
 `CommonJS <https://nodejs.org/docs/latest-v16.x/api/modules.html>`__, with type
 definitions for TypeScript alongside.
 
-This mean that we only support environments (browsers or runtimes) that were
-released after mid-2018 out of the box, if you wish to target these
-environments, then you will need to cross-compile our SDKs via the use of
-`Babel <https://babeljs.io>`__, `webpack <https://webpack.js.org/>`__,
-`SWC <https://swc.rs/>`__, or similar.
+This means that out of the box, we only support environments (browsers or
+runtimes) that were released after mid-2018, if you wish to target other (older)
```

Comment:
I would almost just say "older" instead of but other and older

Reply:
mm, I wanted to emphasise that's it's anything that's not this, not just "older" environments. Maybe `(e.g., older)` ?

Reply:
but what other environments other than the older ones actually apply?

Reply:
I'm not sure of top of head, tbh. I just wanted to be more broad to cover all basis.